app/controllers/users/phones_controller.rb

```diff
@@ -43,5 +44,12 @@ def confirm_phone
         selected_default_number: @new_phone_form.otp_make_default_number
       )
     end
+
+    def check_max_phone_numbers_per_account
+      max_phones_count = IdentityConfig.store.max_phone_numbers_per_account
+      return if current_user.phone_configurations.count < max_phones_count
+      flash[:phone_error] = t('users.phones.error_message')
+      redirect_to account_url(anchor: 'phones')
```

Comment:
I modeled it off of how we setup max emails. 
https://github.com/18F/identity-idp/blob/main/app/controllers/users/emails_controller.rb#L96

Reply:
A difference in that case is we only show email addresses on the "Your account" page, so it makes sense to send the user back there.

I only raise it because I started from the "Your authentication methods" screen, clicked "Add phone number", and was a bit surprised that I was taken to a different page to see the alert.

Ideally we wouldn't have multiple places where we list phone numbers, but that's an issue for another time 🤷

Reply:
Hmm thats a good point. There should be a way to redirect to the page we are.. Rails has a way I am not remembering off the top of my head. I will fix this.

Reply:
`redirect_back`?

Reply:
Sorry, forgot we have some security checks around the use of `redirect_back`, so it would have to be something like:
```suggestion
       redirect_back fallback_location: account_url(anchor: 'phones'), allow_other_host: false
```

Reply:
I was looking at request.referer

Reply:
We'll want to be very careful with referer form a security point-of-view. There's also a `GoBackHelper` module that exists to help with this, maybe we can use it?

Reply:
`redirect_back` uses the referer header, and is probably the method we should call here, along with the options in my suggestion above